app/src/main/java/ar/edu/utn/frba/homeassistant/MainActivity.kt

```diff
@@ -4,44 +4,163 @@ import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
 import androidx.activity.enableEdgeToEdge
+import androidx.compose.foundation.layout.Arrangement
```

Comment:
There are several unused imports, you can quickly remove them using "Optimize imports".
![image](https://github.com/user-attachments/assets/cb5a6c1e-7c19-4f8f-b2d6-bb545f0c3357)